README.md

```diff
@@ -1,40 +1,131 @@
+![OpenUI5 logo](http://openui5.org/images/OpenUI5_new_big_side.png)
+
 # openui5-basic-template-app
-This is a basic app template which you can use as a starting point to build OpenUI5 apps.
+> [OpenUI5](https://github.com/SAP/openui5) basic template app using the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling).
+
+## Prerequisites
+- The **UI5 CLI** of the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling#installing-the-ui5-cli).
```

Comment:
Remove the "-" and put it in one paragraph
The **UI5 build and development tooling command line interface (UI5 CLI)** is installed. For installation instructions please see [Installing the UI5 CLI](https://github.com/SAP/ui5-tooling#installing-the-ui5-cli).